File: 0027-remove-element/0027-remove-element.cs
public class Solution {
    public int RemoveElement(int[] nums, int val) {
      int write = 0;
      int n = nums.Length;
      int count = 0;
      for(int read = 0; read < n; read++){
        if(nums[read] != val){
          nums[write] = nums[read];
          write++;
          
        }

      }
      return write;
     
        
    }
}

File: 0027-remove-element/0027-remove-element.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
      #ie) [3,2,2,3]->[2,2,2,2]-> but k inc if nums[i]!=val
      #ie) 2 != 3 -> 
      k = 0
      for i in range(len(nums)):
        if nums[i] != val:
          nums[k] = nums[i] #do not need to remove it
          k+=1
      return k
        